README.md

```diff
@@ -3,15 +3,17 @@
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.833438.svg)](https://doi.org/10.5281/zenodo.833438)
 [![License](http://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/weecology/portalPredictions/master/LICENSE)
 
+#### [Portal Forecasting Website](http://portal.naturecast.org/)
+
 This is the main repository for predictions made on the Portal rodent census data [Portal Project](http://portal.weecology.org/).
 
-Predictions are made approximately monthly, immediately prior to a new trapping session (trapping occurs as close to each new moon as possible.
+Predictions are made and archived weekly. Approximately once a month, one of these forecasts is made immediately prior to a new trapping session (trapping occurs as close to each new moon as possible).
 
-Portalforecasts.R is the main function used to produce forecasts. Multiple models are used for each new forecast, and are descibed in their respective files. 
+## How to add a new model
 
-Models currently in use are in the models directory. Models that were used or we're developing but not currently using are under DevelopingModels.
+Models currently in use are in the [models](models) directory. All the information needed to make a new forecast can be found in [tools](tools), which is updated weekly before new forecasts are made. This includes rodent and weather data files. `model.yaml` includes information that can be used to ensure that your model's forecast information matches other models forecasting the same data at the same time.
```

Comment:
I like this idea of prepping data files and metadata that models can use instead of the raw data. I think it can be more explicit by creating another folder called something like `data_environment` and putting the data csv's and the model.yaml file in it instead of in the `tools` dir. This would  reinforce the implication that things in `data_environment` is what will be updating every week.